packages/namers/default/src/DefaultNamer.js

```diff
@@ -114,7 +114,7 @@ function nameFromContent(
 
     return path
       .join(path.relative(entryRoot, path.dirname(entryFilePath)), name)
-      .replace(/\.\.(\/|\\)/g, '__$1');
+      .replace(/\.\.(\/|\\)/g, 'up_$1');
```

Comment:
Let me know if you prefer a different name, like 
```suggestion
      .replace(/\.\.(\/|\\)/g, 'dist/$1');
```